src/containers/Dashboard/Dashboard.js

```diff
@@ -3,11 +3,14 @@ import Navbar from "./Navbar";
 import ContentContainer from "../Dashboard/ContentContainer";
 
 class Dashboard extends Component {
+
   render() {
+    const email = localStorage.getItem("email");
```

Comment:
@Humnaa 
Create separate service for Session and get email from there

Reply:
I have created separate folder service and getting session related info form there